buildSrc/src/main/kotlin/knit-conventions.gradle.kts

```diff
@@ -2,19 +2,14 @@ plugins {
     id("kotlinx-knit")
 }
 
+// TODO check that roots are correct
 knit {
     siteRoot = "https://kotlinlang.org/api/kotlinx.coroutines"
     moduleRoots = listOf(".", "integration", "reactive", "ui")
     moduleDocs = "build/dokka/htmlPartial"
     dokkaMultiModuleRoot = "build/dokka/htmlMultiModule/"
 }
 
-tasks.named("knitPrepare").configure {
-    val knitTask = this
-    // In order for knit to operate, it should depend on and collect
-    // all Dokka outputs from each module
-    allprojects {
-        val dokkaTasks = tasks.matching { it.name == "dokkaHtmlMultiModule" }
-        knitTask.dependsOn(dokkaTasks)
-    }
+tasks.named("knitPrepare") {
+    dependsOn("dokka")
```

Comment:
Does the coroutines Gradle config have a task named `dokka`? Or should this be `dokkaGenerate`?